client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/upgrade-popover/style.scss

```diff
@@ -0,0 +1,40 @@
+.upgrade-popover {
+	border-radius: 4px;
+
+	.popover__inner {
+		display: flex;
+		gap: 16px;
+		padding: 16px;
+		flex-direction: column;
+		align-items: flex-start;
+	}
+}
+
+.upgrade-popover__heading {
+	font-size: rem(14px);
+	line-height: 1.5;
+}
+
+.upgrade-popover__list {
+	margin: 0;
+	list-style: none;
+	font-size: rem(14px);
+	text-align: left;
+	display: flex;
+	flex-direction: column;
+	gap: 2px;
+
+	li {
+		background: url(calypso/assets/images/jetpack/jetpack-green-checkmark.svg) no-repeat 0 2px;
```

Comment:
This file path needs to be enclosed in quotes, doesn't it? Or am I learning something new today? (Either could be true! 😁)

Reply:
Both formats are valid.